scrapers.js

```diff
@@ -2134,6 +2134,86 @@ const scrapers = [
id\":\"BS\",\"name\":\"Bahamas\"},{\"uid\":\"VU\",\"name\":\"Vanuatu\"},{\"uid\":\"FK\",\"name\":\"Ilhas Malvinas\"},{\"uid\":\"GM\",\"name\":\"Gâmbia\"},{\"uid\":\"QA\",\"name\":\"Catar\"},{\"uid\":\"JM\",\"name\":\"Jamaica\"},{\"uid\":\"CY\",\"name\":\"Chipre\"},{\"uid\":\"PR\",\"name\":\"Porto Rico\"},{\"uid\":\"PS\",\"name\":\"Palestina\"},{\"uid\":\"BN\",\"name\":\"Brunei\"},{\"uid\":\"TT\",\"name\":\"Trinidad e Tobago\"},{\"uid\":\"CV\",\"name\":\"Cabo Verde\"},{\"uid\":\"PF\",\"name\":\"Polinésia Francesa\"},{\"uid\":\"WS\",\"name\":\"Samoa\"},{\"uid\":\"LU\",\"name\":\"Luxemburgo\"},{\"uid\":\"RE\",\"name\":\"Ilha da Reunião\"},{\"uid\":\"KM\",\"name\":\"Comores\"},{\"uid\":\"MU\",\"name\":\"Maurício\"},{\"uid\":\"FO\",\"name\":\"Ilhas Faroé\"},{\"uid\":\"MQ\",\"name\":\"Martinica\"},{\"uid\":\"ST\",\"name\":\"São Tomé e Príncipe\"},{\"uid\":\"AN\",\"name\":\"Antilhas Neerlandesas\"},{\"uid\":\"DM\",\"name\":\"Dominica\"},{\"uid\":\"GP\",\"name\":\"Guadalupe\"},{\"uid\":\"TO\",\"name\":\"Tonga\"},{\"uid\":\"KI\",\"name\":\"Quiribati\"},{\"uid\":\"FM\",\"name\":\"Micronésia\"},{\"uid\":\"BH\",\"name\":\"Bahrein\"},{\"uid\":\"AD\",\"name\":\"Andorra\"},{\"uid\":\"MP\",\"name\":\"Ilhas Marianas do Norte\"},{\"uid\":\"PW\",\"name\":\"Palau\"},{\"uid\":\"SC\",\"name\":\"Seicheles\"},{\"uid\":\"AG\",\"name\":\"Antígua e Barbuda\"},{\"uid\":\"BB\",\"name\":\"Barbados\"},{\"uid\":\"TC\",\"name\":\"Turks e Caicos\"},{\"uid\":\"VC\",\"name\":\"São Vicente e Granadinas\"},{\"uid\":\"LC\",\"name\":\"Santa Lúcia\"},{\"uid\":\"YT\",\"name\":\"Mayotte\"},{\"uid\":\"VI\",\"name\":\"Ilhas Virgens Americanas\"},{\"uid\":\"GD\",\"name\":\"Granada\"},{\"uid\":\"MT\",\"name\":\"Malta\"},{\"uid\":\"MV\",\"name\":\"Maldivas\"},{\"uid\":\"KY\",\"name\":\"Ilhas Cayman\"},{\"uid\":\"KN\",\"name\":\"São Cristóvão e Névis\"},{\"uid\":\"MS\",\"name\":\"Montserrat\"},{\"uid\":\"NU\",\"name\":\"Niue\"},{\"uid\":\"PM\",\"name\":\"São Pedro e Miquelão\"},{\"uid\":\"CK\",\"name\":\"Ilhas Cook\"},{\"uid\":\"WF\",\"name\":\"Wallis e Futuna\"},{\"uid\":\"AS\",\"name\":\"Samoa Americana\"},{\"uid\":\"MH\",\"name\":\"Ilhas Marshall\"},{\"uid\":\"AW\",\"name\":\"Aruba\"},{\"uid\":\"LI\",\"name\":\"Liechtenstein\"},{\"uid\":\"VG\",\"name\":\"Ilhas Virgens Britânicas\"},{\"uid\":\"SH\",\"name\":\"Santa Helena\"},{\"uid\":\"JE\",\"name\":\"Jersey\"},{\"uid\":\"AI\",\"name\":\"Anguilla\"},{\"uid\":\"GG\",\"name\":\"Guernsey\"},{\"uid\":\"SM\",\"name\":\"San Marino\"},{\"uid\":\"BM\",\"name\":\"Bermudas\"},{\"uid\":\"TV\",\"name\":\"Tuvalu\"},{\"uid\":\"NR\",\"name\":\"Nauru\"},{\"uid\":\"GI\",\"name\":\"Gibraltar\"},{\"uid\":\"PN\",\"name\":\"Ilhas Pitcairn\"},{\"uid\":\"MC\",\"name\":\"Mônaco\"},{\"uid\":\"VA\",\"name\":\"Vaticano\"},{\"uid\":\"IM\",\"name\":\"Ilha de Man\"},{\"uid\":\"GU\",\"name\":\"Guam\"},{\"uid\":\"SG\",\"name\":\"Singapura\"},{\"uid\":\"SS\",\"name\":\"Sudão do Sul\"},{\"uid\":\"SX\",\"name\":\"São Martinho\"},{\"uid\":\"BL\",\"name\":\"São Bartolomeu\"}]"),
```

Comment:
Can you put the parsed result here instead of the string?
```suggestion
    _dataIds: [{"uid":11,"name":"Rondônia"},{"uid":12,"name":"Acre"},{"uid":13,"name":"Amazonas"},{"uid":14,"name":"Roraima"},{"uid":15,"name":"Pará"},{"uid":16,"name":"Amapá"},{"uid":17,"name":"Tocantins"},{"uid":21,"name":"Maranhão"},{"uid":22,"name":"Piauí"},{"uid":23,"name":"Ceará"},{"uid":24,"name":"Rio Grande do Norte"},{"uid":25,"name":"Paraíba"},{"uid":26,"name":"Pernambuco"},{"uid":27,"name":"Alagoas"},{"uid":28,"name":"Sergipe"},{"uid":29,"name":"Bahia"},{"uid":31,"name":"Minas Gerais"},{"uid":32,"name":"Espírito Santo"},{"uid":33,"name":"Rio de Janeiro"},{"uid":35,"name":"São Paulo"},{"uid":41,"name":"Paraná"},{"uid":42,"name":"Santa Catarina"},{"uid":43,"name":"Rio Grande do Sul"},{"uid":50,"name":"Mato Grosso do Sul"},{"uid":51,"name":"Mato Grosso"},{"uid":52,"name":"Goiás"},{"uid":53,"name":"Distrito Federal"},{"uid":"PE","name":"Peru"},{"uid":"GF","name":"Guiana Francesa"},{"uid":"BF","name":"Burkina Faso"},{"uid":"FR","name":"França"},{"uid":"LY","name":"Libéria"},{"uid":"BY","name":"Belarus"},{"uid":"PK","name":"Paquistão"},{"uid":"ID","name":"Indonésia"},{"uid":"YE","name":"Iêmen"},{"uid":"MG","name":"Madagascar"},{"uid":"BO","name":"Bolívia"},{"uid":"CI","name":"Costa do Marfim"},{"uid":"DZ","name":"Argélia"},{"uid":"CH","name":"Suíça"},{"uid":"CM","name":"Cameroun"},{"uid":"MK","name":"Macedônia do Norte"},{"uid":"BW","name":"Botsuana"},{"uid":"UA","name":"Ucrânia"},{"uid":"KE","name":"Quênia"},{"uid":"TW","name":"Taiwan"},{"uid":"JO","name":"Jordânia"},{"uid":"MX","name":"México"},{"uid":"AE","name":"Emirados Árabes Unidos"},{"uid":"BZ","name":"Belize"},{"uid":"BR","name":"Brasil"},{"uid":"SL","name":"Serra Leoa"},{"uid":"ML","name":"Mali"},{"uid":"CD","name":"República Democrática do Congo"},{"uid":"IT","name":"Itália"},{"uid":"SO","name":"Somália"},{"uid":"AF","name":"Afeganistão"},{"uid":"BD","name":"Bangladesh"},{"uid":"DO","name":"República Dominicana"},{"uid":"GW","name":"Guiné-Bissau"},{"uid":"GH","name":"Gana"},{"uid":"AT","name":"Áustria"},{"uid":"SE","name":"Suécia"},{"uid":"TR","name":"Turquia"},{"uid":"UG","name":"Uganda"},{"uid":"MZ","name":"Moçambique"},{"uid":"JP","name":"Japão"},{"uid":"NZ","name":"Nova Zelândia"},{"uid":"CU","name":"Cuba"},{"uid":"VE","name":"Venezuela"},{"uid":"PT","name":"Portugal"},{"uid":"CO","name":"Colômbia"},{"uid":"MR","name":"Mauritânia"},{"uid":"AO","name":"Angola"},{"uid":"DE","name":"Alemanha"},{"uid":"SD","name":"Sudão"},{"uid":"TH","name":"Tailândia"},{"uid":"AU","name":"Austrália"},{"uid":"PG","name":"Papua Nova Guiné"},{"uid":"IQ","name":"Iraque"},{"uid":"HR","name":"Croácia"},{"uid":"GL","name":"Groelândia"},{"uid":"NE","name":"Níger"},{"uid":"DK","name":"Dinamarca"},{"uid":"LV","name":"Letônia"},{"uid":"RO","name":"Romênia"},{"uid":"ZM","name":"Zâmbia"},{"uid":"IR","name":"Irã"},{"uid":"MM","name":"Myanmar"},{"uid":"ET","name":"Etiópia"},{"uid":"GT","name":"Guatemala"},{"uid":"SR","name":"Suriname"},{"uid":"EH","name":"Saara Ocidental"},{"uid":"CZ","name":"República Tcheca"},{"uid":"TD","name":"Chade"},{"uid":"AL","name":"Albânia"},{"uid":"FI","name":"Finlândia"},{"uid":"SY","name":"Síria"},{"uid":"KG","name":"Quirguistão"},{"uid":"SB","name":"Ilhas Salomão"},{"uid":"OM","name":"Omã"},{"uid":"PA","name":"Panamá"},{"uid":"AR","name":"Argentina"},{"uid":"GB","name":"Reino Unido"},{"uid":"CR","name":"Costa Rica"},{"uid":"PY","name":"Paraguai"},{"uid":"GN","name":"Guiné"},{"uid":"IE","name":"Irlanda"},{"uid":"NG","name":"Nigéria"},{"uid":"TN","name":"Tunísia"},{"uid":"PL","name":"Polônia"},{"uid":"NA","name":"Namíbia"},{"uid":"ZA","name":"África do Sul"},{"uid":"EG","name":"Egito"},{"uid":"TZ","name":"Tanzânia"},{"uid":"GE","name":"Geórgia"},{"uid":"SA","name":"Arábia Saudita"},{"uid":"VN","name":"Vietnã"},{"uid":"RU","name":"Rússia"},{"uid":"HT","name":"Haiti"},{"uid":"BA","name":"Bósnia e Herzegovina"},{"uid":"IN","name":"Índia"},{"uid":"CN","name":"China"},{"uid":"CA","name":"Canadá"},{"uid":"SV","name":"El Salvador"},{"uid":"GY","name":"Guiana"},{"uid":"BE","name":"Bélgica"},{"uid":"GQ","name":"Guiné Equatorial"},{"uid":"LS","name":"Lesoto"},{"uid":"BG","name":"Bulgária"},{"uid":"BI","name":"Burundi"},{"uid":"DJ","name":"Djibouti"},{"uid":"AZ","name":"Azerbaijão"},{"uid":"MY","name":"Malásia"},{"uid":"PH","name":"Filipinas"},{"uid":"UY","name":"Uruguai"},{"uid":"CG","name":"República Democrática do Congo"},{"uid":"RS","name":"Sérvia"},{"uid":"ME","name":"Montenegro"},{"uid":"EE","name":"Estônia"},{"uid":"RW","name":"Ruanda"},{"uid":"AM","name":"Armênia"},{"uid":"SN","name":"Senegal"},{"uid":"TG","name":"Togo"},{"uid":"ES","name":"Espanha"},{"uid":"GA","name":"Gabão"},{"uid":"HU","name":"Hungria"},{"uid":"MW","name":"Malawi"},{"uid":"TJ","name":"Tajiquistão"},{"uid":"KH","name":"Camboja"},{"uid":"KR","name":"Coreia do Sul"},{"uid":"HN","name":"Honduras"},{"uid":"IS","name":"Islândia"},{"uid":"NI","name":"Nicarágua"},{"uid":"CL","name":"Chile"},{"uid":"MA","name":"Marrocos"},{"uid":"LR","name":"Libéria"},{"uid":"NL","name":"Holanda"},{"uid":"CF","name":"República Centro-Africana"},{"uid":"SK","name":"Eslováquia"},{"uid":"LT","name":"Lituânia"},{"uid":"ZW","name":"Zimbábue"},{"uid":"LK","name":"Sri Lanka"},{"uid":"IL","name":"Israel"},{"uid":"LA","name":"Laos"},{"uid":"KP","name":"Coreia do Norte"},{"uid":"GR","name":"Grécia"},{"uid":"TM","name":"Turcomenistão"},{"uid":"EC","name":"Equador"},{"uid":"BJ","name":"Benin"},{"uid":"SI","name":"Eslovênia"},{"uid":"NO","name":"Noruega"},{"uid":"MD","name":"Moldávia"},{"uid":"LB","name":"Líbano"},{"uid":"NP","name":"Nepal"},{"uid":"ER","name":"Eritreia"},{"uid":"US","name":"Estados Unidos"},{"uid":"KZ","name":"Cazaquistão"},{"uid":"SZ","name":"Suazilândia"},{"uid":"UZ","name":"Uzbequistão"},{"uid":"MN","name":"Mongólia"},{"uid":"BT","name":"Butão"},{"uid":"NC","name":"Nova Caledônia"},{"uid":"FJ","name":"Fiji"},{"uid":"KW","name":"Kuwait"},{"uid":"TL","name":"Timor-Leste"},{"uid":"BS","name":"Bahamas"},{"uid":"VU","name":"Vanuatu"},{"uid":"FK","name":"Ilhas Malvinas"},{"uid":"GM","name":"Gâmbia"},{"uid":"QA","name":"Catar"},{"uid":"JM","name":"Jamaica"},{"uid":"CY","name":"Chipre"},{"uid":"PR","name":"Porto Rico"},{"uid":"PS","name":"Palestina"},{"uid":"BN","name":"Brunei"},{"uid":"TT","name":"Trinidad e Tobago"},{"uid":"CV","name":"Cabo Verde"},{"uid":"PF","name":"Polinésia Francesa"},{"uid":"WS","name":"Samoa"},{"uid":"LU","name":"Luxemburgo"},{"uid":"RE","name":"Ilha da Reunião"},{"uid":"KM","name":"Comores"},{"uid":"MU","name":"Maurício"},{"uid":"FO","name":"Ilhas Faroé"},{"uid":"MQ","name":"Martinica"},{"uid":"ST","name":"São Tomé e Príncipe"},{"uid":"AN","name":"Antilhas Neerlandesas"},{"uid":"DM","name":"Dominica"},{"uid":"GP","name":"Guadalupe"},{"uid":"TO","name":"Tonga"},{"uid":"KI","name":"Quiribati"},{"uid":"FM","name":"Micronésia"},{"uid":"BH","name":"Bahrein"},{"uid":"AD","name":"Andorra"},{"uid":"MP","name":"Ilhas Marianas do Norte"},{"uid":"PW","name":"Palau"},{"uid":"SC","name":"Seicheles"},{"uid":"AG","name":"Antígua e Barbuda"},{"uid":"BB","name":"Barbados"},{"uid":"TC","name":"Turks e Caicos"},{"uid":"VC","name":"São Vicente e Granadinas"},{"uid":"LC","name":"Santa Lúcia"},{"uid":"YT","name":"Mayotte"},{"uid":"VI","name":"Ilhas Virgens Americanas"},{"uid":"GD","name":"Granada"},{"uid":"MT","name":"Malta"},{"uid":"MV","name":"Maldivas"},{"uid":"KY","name":"Ilhas Cayman"},{"uid":"KN","name":"São Cristóvão e Névis"},{"uid":"MS","name":"Montserrat"},{"uid":"NU","name":"Niue"},{"uid":"PM","name":"São Pedro e Miquelão"},{"uid":"CK","name":"Ilhas Cook"},{"uid":"WF","name":"Wallis e Futuna"},{"uid":"AS","name":"Samoa Americana"},{"uid":"MH","name":"Ilhas Marshall"},{"uid":"AW","name":"Aruba"},{"uid":"LI","name":"Liechtenstein"},{"uid":"VG","name":"Ilhas Virgens Britânicas"},{"uid":"SH","name":"Santa Helena"},{"uid":"JE","name":"Jersey"},{"uid":"AI","name":"Anguilla"},{"uid":"GG","name":"Guernsey"},{"uid":"SM","name":"San Marino"},{"uid":"BM","name":"Bermudas"},{"uid":"TV","name":"Tuvalu"},{"uid":"NR","name":"Nauru"},{"uid":"GI","name":"Gibraltar"},{"uid":"PN","name":"Ilhas Pitcairn"},{"uid":"MC","name":"Mônaco"},{"uid":"VA","name":"Vaticano"},{"uid":"IM","name":"Ilha de Man"},{"uid":"GU","name":"Guam"},{"uid":"SG","name":"Singapura"},{"uid":"SS","name":"Sudão do Sul"},{"uid":"SX","name":"São Martinho"},{"uid":"BL","name":"São Bartolomeu"}],
```